compiler/next/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Hewlett Packard Enterprise Development LP
+# Copyright 2022 Hewlett Packard Enterprise Development LP
```

Comment:
Everything compiler/next should change `2021` to `2021-2022`

Reply:
Ah sheesh, how did I miss this.  Thanks Lydia!  More to the point, everything that changed from being simply 2021 should change to 2021-2022 other than the little file that people are meant to use to add a new copyright message.